src/locales/fr/translation.json

```diff
@@ -1,4 +1,19 @@
 {
+    "Sort notes by": "Trier les notes",
+    "Theme": "Thème",
+    "Notebooks in drawer": "Bloc-notes dans le menu",
+    "Spaces per indent": "Espaces pour indenter",
+    "Text editor": "Éditeur de texte",
+    "Your key": "Votre clé",
+    "Change passphrase": "Changer la phrase de passe",
```

Comment:
"passphrase" == "phrase secr`ete" or "mot de passe" (I do not have the accent on this but you got it :wink: )